ui/src/main/javascript/app/lobby/lobby.tpl.html

```diff
@@ -1,26 +1,46 @@
-<div class="logo">
+<div class="lobby-game-section">
+    <div class="sw-box box">
+        <form class ="sw-align-form">
+            <div class="sw-box-1">
+                <span>
+                    <label for="name" class="sw-title sw-label">Name Game</label>
+                    <input class="sw-input" id="name" ng-model="gameroom.name" required placeholder="Name">
+                    </input>
```

Comment:
Remove this tag.
Use: <input  id="name" class="sw-input" ng-model="gameroom.name" required placeholder="Name" />